### PR TITLE
feat(toml): update consensus constants for Testnet8

### LINKED ISSUE #1175: [T8] Decide consensus constants
Discussion about the consensus constants and configurable parameters that we want to test in Testnet 8

### PR DESCRIPTION
**Pull request by lrubiorod**
Close #1175

### CODE REVIEW

**Comment by lrubiorod** [comment]
@aesedepece, please include a comment with the genesis_hash when it would be created

File: witnet.toml [hunk 1]
```diff
@@ -52,10 +52,12 @@ server_address = "127.0.0.1:21338"
 
 [consensus_constants]
 checkpoints_period = 45
-checkpoint_zero_timestamp = 1585742400
+# Monday, 27-Apr-2020, 14:00 UTC
+checkpoint_zero_timestamp = 1587996000
 bootstrap_hash = "00000000000000000000000000000000000000007769746e65742d302e372e33"
```
Comment (aesedepece) [inline witnet.toml:57]:
```suggestion
bootstrap_hash = "00000000000000000000000000000000000000007769746e65742d302e382e30"
```

File: witnet.toml [hunk 2]
```diff
@@ -52,10 +52,12 @@ server_address = "127.0.0.1:21338"
...
 [consensus_constants]
 checkpoints_period = 45
-checkpoint_zero_timestamp = 1585742400
-bootstrap_hash = "00000000000000000000000000000000000000007769746e65742d302e372e33"
-genesis_hash = "15493414fde678612be9811777bb26f019af75de23daffd41974fccd784c89b2"
-mining_replication_factor = 1
+# Monday, 27-Apr-2020, 14:00 UTC
+checkpoint_zero_timestamp = 1587996000
+bootstrap_hash = "00000000000000000000000000000000000000007769746e65742d302e382e30"
+genesis_hash = "a6901661b32aed279d5c511b5edf0febb0f068927f533619271802fa49e842eb"
```
Comment (aesedepece) [inline witnet.toml:58]: Hey let's not PR and merge this `genesis_hash`, which is intended only for local, testing purposes. I'd rather do another PR for the final `genesis_hash` when we know it.
